packages/api/src/routes/settings.ts

```diff
@@ -114,6 +155,28 @@ router.get(
 const updateSettingsSchema = z
   .object({
     webhookUrl: z.string().url().max(maxWebhookUrlLength).or(z.literal("").nullable().optional()),
+    mrFilters: z
+      .array(
+        z.object({
+          key: z.enum(mrSectionsKeys),
+          resourceType: z.enum(mrSectionsResourceTypes),
+          dateFilter: z
+            .object({
+              from: z
+                .string()
+                .refine(v => dayjs(v, ISO_DATE, true).isValid())
+                .optional(),
+              to: z
+                .string()
+                .refine(v => dayjs(v, ISO_DATE, true).isValid())
+                .optional(),
+            })
+            .optional(),
+          stringFilter: z.string().optional(),
+        })
+      )
+      .min(mrSectionsKeys.length)
```

Comment:
We want to force every one to be defined?

Reply:
I think we should if not. it will remove the section

Reply:
I think we should refactor it once we get this out the door so that if it's not there, we simply take the default filters for the section. That, or we should intentionally allow CX's to remove sections by not having filters for them.